File: TellDontAsk/PayrollSystem/Employee.cs
 using System;
using System.Collections.Generic;

namespace PayrollSystem
{
	public class Employee
	{
    public void Save()
    {
      Console.WriteLine("Saving");
    }

    public void SetBonusForYear(int i, decimal oneTimeBonus)
    {
      Console.WriteLine(string.Format("Setting for {0} to {1}", i, oneTimeBonus));
    }

    public int GetYearseOfService()
    {
      return 123;
    }

    public void SetSalary(decimal newSalary)
    {
      Console.WriteLine("New Salary is: " + newSalary);
    }

    public PayGrade GetPayGrade()
    {
      return new PayGrade();
    }

    public decimal GetSalary()
    {
      return new decimal(44.4);
    }
	}


}

File: Java/PayrollSystem/src/main/java/Employee.java
/**
 * Created by astral on 24.03.15.
 */
public class Employee
{
  public void Save()
  {
    System.out.println("Saving");
  }

  public void SetBonusForYear(int i, int oneTimeBonus)
  {
    System.out.printf("Setting for %d to %d \n", i, oneTimeBonus);
  }

  public int GetYearsOfService()
  {
    return 123;
  }

  public void SetSalary(int newSalary)
  {
    System.out.println("New Salary is: " + newSalary);
  }

  public PayGrade GetPayGrade()
  {
    return new PayGrade();
  }

  public int GetSalary()
  {
    return 44;
  }
}